### ISSUE TITLE
Fix Request for AvoidHardcodedConnectionConfig - false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#417]
```
class Foo {
    
    private final int readTimeout; // <--- false positive

    Foo() {
        readTimeout = Configuration.getReadTimeout(); 
    }
}
```
